go.mod

```diff
@@ -3,7 +3,10 @@ module github.com/kyma-project/cli
 go 1.18
 
 // required to avoid locking with gardner dependency updates
-replace k8s.io/client-go => k8s.io/client-go v0.24.4
+replace (
+	github.com/moby/moby => github.com/moby/moby v20.10.18+incompatible
```

Comment:
This replace statement is not needed. Just replacing the version on lines 16 and 141 is enough.

Reply:
Done.